server/services/store/sqlstore/board.go

```diff
@@ -680,21 +680,35 @@ func (s *SQLStore) searchBoardsForUser(db sq.BaseRunner, term, userID string, in
 	}
 
 	if term != "" {
-		// break search query into space separated words
-		// and search for all words.
-		// This should later be upgraded to industrial-strength
-		// word tokenizer, that uses much more than space
-		// to break words.
-
-		conditions := sq.And{}
-
-		for _, word := range strings.Split(strings.TrimSpace(term), " ") {
-			conditions = append(conditions, sq.Like{"lower(b.title)": "%" + strings.ToLower(word) + "%"})
+		if searchField == model.BoardSearchFieldPropertyName {
+			switch s.dbType {
+			case model.PostgresDBType:
+				where := "b.properties->? is not null"
+				query = query.Where(where, term)
+			case model.MysqlDBType:
+				where := "JSON_EXTRACT(b.properties, ?) IS NOT NULL"
+				query = query.Where(where, "$."+term)
+			default:
+				where := "b.properties LIKE ?"
+				query = query.Where(where, "%\""+term+"\"%")
```

Comment:
Shouldn't we be able to use something like this here for SQLite?

```suggestion
				where := "json_extract(b.properties, '$.?') IS NOT NULL"
				query = query.Where(where, term)
```

Reply:
Thanks @mgdelacroix.  I really thought `json_extract` wasn't available until a later version of Sqlite than we support as minimum version.  Now I think I was looking at the [-> and ->> operators](https://www.sqlite.org/json1.html#jex) that require 3.38.0.   My distro comes with 3.31.1